src/style/color.js

```diff
@@ -0,0 +1,55 @@
+// Colors
+export const ColorGrey = '#f7f7f7'
```

Comment:
Refactor the colors to contain only the effectively used once and delete the rest.